tests/main/user-session-env/task.yaml

```diff
@@ -41,6 +41,11 @@ restore: |
             # The user-runtime-dir@.service unit did not exist before
             # Systemd 239, so ignore errors.
             systemctl stop "user-runtime-dir@$uid.service" || true
+
+            # Ensure runtime dir is cleaned up on older systems
```

Comment:
Can you clarify this comment. What are older systems and why are they not cleaned up?